CHANGELOG.md

```diff
@@ -52,6 +52,7 @@ We use *breaking :warning:* to mark changes that are not backward compatible (re
 - [#6519](https://github.com/thanos-io/thanos/pull/6519) Reloader: Use timeout for initial apply.
 - [#6509](https://github.com/thanos-io/thanos/pull/6509) Store Gateway: Remove `memWriter` from `fileWriter` to reduce memory usage when sync index headers.
 - [#6556](https://github.com/thanos-io/thanos/pull/6556) Thanos compact: respect block-files-concurrency setting when downsampling
+- [#6539](https://github.com/thanos-io/thanos/pull/6539) Store : Changed `--sync-block-duration` default 3m to 15m.
```

Comment:
Is this potentially a breaking change? Should we add a warning / breaking change note in the changelog? cc @douglascamata @saswatamcode

Reply:
Definitely worth the warning of the different default value for those not already changing it. I wouldn't consider it a breaking change though, as changing this default value shouldn't truly break things.

Reply:
sure 👍